src/utils/logging.js

```diff
@@ -88,8 +100,7 @@ const logToSentry = (event: string | Error, level: SeverityType, extras: Extras)
   }
 
   const tags = {
-    ...getServerVersionTags(),
-    // Other tags go here; they're useful for event aggregation. See
+    // Tags can go here; they're useful for event aggregation. See
     // https://docs.sentry.io/platforms/javascript/enriching-events/tags/.
   };
```

Comment:
This should probably have a comment pointing to where we actually do set the tags.

Or better yet: once that `setTags` call moves back into this file (per other review-comment above), this and the `setTags` call below can be deleted. It's possible that in the future we'll have some tags that apply only in the explicit-logging case, but I think the ideal is that we won't and instead all tags will go onto scopes so that they apply equally to uncaught exceptions.